R/tidy.R

```diff
@@ -208,16 +205,17 @@ unmask_source = function(text.mask, spaces) {
 
 #' Format all R scripts under a directory, or specified R scripts
 #'
-#' \code{tidy_dir()} first looks for all the R scripts under a directory (using
-#' the pattern \code{"[.][RrSsQq]$"}), then uses \code{\link{tidy_source}()} to
-#' tidy these scripts. The original scripts will be overwritten with reformatted
-#' code if reformatting was successful. You may need to back up the original
-#' directory first if you do not fully understand the tricks used by
-#' \code{\link{tidy_source}()}. \code{tidy_file()} formats specified R scripts.
-#' @param path the directory
-#' @param recursive whether to recursively look for R scripts under \code{path}
-#' @param ... other arguments to be passed to \code{\link{tidy_source}()}
-#' @param file a vector of filenames
+#' Look for all R scripts under a directory (using the pattern \code{"[.][RrSsQq]$"}),
+#' then tidy them with \code{\link{tidy_source}()}.
+#' If successful, the original scripts will be overwritten with reformatted
+#' ones. Back up the original directory first if you do not fully understand the
+#' tricks used by \code{\link{tidy_source}()}.
+#' \code{tidy_file()} formats scripts specified by file names.
+#'
+#' @param path The path to a directory containning R scripts.
+#' @param recursive Whether to recursively look for R scripts under \code{path}.
+#' @param ... Other arguments to be passed to \code{\link{tidy_source}()}.
+#' @param file A vector of file names.
```

Comment:
Both `filename` and `file name` are okay: https://en.wikipedia.org/wiki/Filename